index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="style.css">
+    <title>gabriel's portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <a class="logo-container" href="">
+                <img src="images/image_geometry_1logo.png" alt="logo">
+            </a>
+
+            <nav>
+                <a class="a-logo" href=""><i class="fas fa-bars"></i></a>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1>Gabriel <br> Nwachukwu</h1>
+            </div>
+            <div class="main">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+
+            </div>
+            <button class="collaboration" type="button">
+                start collaboration
```

Comment:
### OPTIONAL

- [ ] The button should and  the button styling: Active, Hover and Pressed - [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)